src/scripts/OSFramework/OSUI/Pattern/Video/Video.ts

```diff
@@ -0,0 +1,344 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+namespace OSFramework.OSUI.Patterns.Video {
+	/**
+	 * Defines the interface for OutSystemsUI Patterns
+	 *
+	 * @export
+	 * @class Video
+	 * @extends {AbstractPattern<VideoConfig>}
+	 * @implements {IVideo}
+	 */
+	export class Video extends AbstractPattern<VideoConfig> implements IVideo {
+		// Store the platform events
+		private _platformEventOnStateChanged: Callbacks.OSOnStateChangedEvent;
+		// Store the video element
+		private _videoElement: HTMLVideoElement;
+		// Store the source element
+		private _videoSourceElement: HTMLSourceElement;
+		// Store the platform events
+
+		constructor(uniqueId: string, configs: JSON) {
+			super(uniqueId, new VideoConfig(configs));
+		}
+
+		// Method that will set the Autoplay attribute
+		private _setAutoplay(): void {
+			if (this.configs.Autoplay) {
+				this._videoElement.autoplay = true;
+
+				if (!this.configs.Muted) {
+					console.warn(
+						`Some modern browsers will not autoplay your video on ${GlobalEnum.PatternName.Video} if it's not muted. The general rule of many browsers is that a user must opt-in to certain actions before they can happen. Set the Muted parameter to True, to start the video automatically.`
+					);
+				}
+			} else {
+				this._videoElement.autoplay = false;
+			}
+		}
+
+		// Method that will set the controls attribute
+		private _setControls(): void {
+			if (this.configs.Controls) {
+				this._videoElement.controls = true;
+			} else {
+				this._videoElement.controls = false;
+			}
+		}
+
+		// Method that will set the Height attribute
+		private _setHeight(): void {
+			if (this.configs.Height !== Constants.EmptyString) {
+				OSUI.Helper.Dom.Attribute.Set(
+					this._videoElement,
+					Patterns.Video.Enum.VideoAttributes.Height,
+					this.configs.Height
+				);
+			} else {
+				OSUI.Helper.Dom.Attribute.Remove(this._videoElement, Patterns.Video.Enum.VideoAttributes.Height);
+			}
+		}
+
+		// Method that will set the loop attribute
+		private _setLoop(): void {
+			if (this.configs.Loop) {
+				this._videoElement.loop = true;
```

Comment:
Same as above

Reply:
Well, here I agree with @BenOsodrac, in these cases we can use the **this._videoElement.loop = this.configs.Loop** instead of being using a boolean validation, right?

Reply:
Sorry but didn't understand the question and make some confusion about that. That is all possible and I will apply the improvement.